sections/advanced/theming.md

```diff
@@ -97,30 +97,29 @@ If you ever need to use the current theme outside styled components (e.g. inside
 the `withTheme` higher order component.
 
 ```jsx
-import { withTheme } from 'styled-components';
+import { withTheme } from 'styled-components'
 
 class MyComponent extends React.Component {
   render() {
-    console.log('Current theme: ', this.props.theme);
+    console.log('Current theme: ', this.props.theme)
     // ...
   }
 }
 
-export default withTheme(MyComponent);
+export default withTheme(MyComponent)
 ```
 
```

Comment:
I'm not sure why you removed these docs, they're valid. `useTheme` is a separate API.

Reply:
It was a @el1f 's suggestion 

> This is off to a great start @arthurfalcao! Thanks!
> Can you please try your hand at adding a couple of words about the hook on the theming section too though? There is a paragraph about accessing the theme using the `useContext` hook to get the theme, I think it could be integrated or even replaced by a mention of `useTheme`.

Reply:
I'm probably mistaken but I feel like in the presence of the `useTheme` hook there aren't many use cases where the user's want to access the theme via the more verbose `useContext` hook. What do you think about it @probablyup ?

Reply:
On second though I realize that users still using v4 could probably need the docs to keep mentioning the `useContext` method. Didn't really think about that possibility before... My bad.